luaui/configs/keyboard_layouts.lua

```diff
@@ -19,28 +30,41 @@ local keyLayouts = {
 			[4] = "V",
 			[5] = "B",
 			[6] = "N",
+			[7] = "M",
 		}
 	},
 	dvorak = {
 		[3] = {
```

Comment:
Why are the rows incomplete?

Reply:
Only keys that are used need to be defined

Reply:
`[` and `]` are used, no? Also, what if someone defines his/her own keys that are not part of this map?

Reply:
The widget concerns only to what it binds and unbinds, it does not fix external keys

Reply:
We could define what [ and ] is positionally but even in qwerty keyboards those keys are not consistently placed

Reply:
Yeah [ ] is totally regional :/